### ISSUE TITLE
JQ false positive warning

### ISSUE THREAD

**Issue by felipecassiors** [koalaman/shellcheck#1949]
The following statement works fine:
```sh
jq --arg v "$PRJNAME" '.dev.projects[$v]'
```
Shellcheck doesn't complain about the `$v` because it is a `jq` argument and not from shell.

However, if I use:

```sh
JQ="jq"
$JQ --arg v "$PRJNAME" '.dev.projects[$v]'
``` 

It complains about [SC2016](https://github.com/koalaman/shellcheck/wiki/SC2016), because it probably doesn't recognize the `jq` command.
